### DISCUSSION TITLE
Loosing GPS Signal

### DISCUSSION THREAD

**Discussion by Rogerllm** [osmandapp/OsmAnd#14284] [Q&A]
I have seen that at some point OSMAND+ is losing the GPS Signal.
In my case, i was hiking, and the app (Android) was not able to get my GPS position.

I was recording the route with a Mi Band 5 (i think it uses gmaps).
There was no data/telephone connection so, gmaps was not able to draw the map, but it was positioning me.
At the end of the track, when i had data again, the track was correctly recorded by the mi band 5 (so the signal was not lost)
But OSMAnd was no able to position me in the map for a long period.
It was a freaky/scary experience as it was my  "off-line" map where i has loaded he route from wikiloc.

Note: I tried maps.me too, loading he downloaded route and it was not able to position me either (i think both aps uses OSM, not sure if this can be the source of the issue).

**Comment by scaidermern** [comment]
> Note: I tried maps.me too, loading he downloaded route and it was not able to position me either (i think both aps uses OSM, not sure if this can be the source of the issue).

The source of the map data really can't be the reason for loosing the GPS signal.

Check with other apps like GPSTest or SatStat. If they still have a signal then this is an issue with these particular apps (but not with OSM). If they have the same problem then there is something wrong with either your device or operating system.
